pmd-visualforce/etc/grammar/VfParser.jjt

```diff
@@ -255,8 +260,9 @@ PARSER_END(VfParser)
 
 <HtmlScriptContentState> TOKEN :
 {
-		 <HTML_SCRIPT_CONTENT: (~[]) >
-		| <HTML_SCRIPT_END_TAG : "</script>" > : AfterTagState
+		  <HTML_SCRIPT_END_TAG : "</script>" > : AfterTagState
+		| <EL_EXPRESSION_IN_SCRIPT: "{!" (<WHITESPACES>)? > : ElInScriptState
```

Comment:
@sgorbaty how would such parser solve this issue? `{!myVar}` could be either valid JS or valid EL... Even more with JS not needing to declare variables.